src/SurveyDesign.jl

```diff
@@ -178,75 +179,158 @@ end
     StratifiedSample <: AbstractSurveyDesign
 
     Survey design sampled by stratification.
+
```

Comment:
I think the doc should have an example.

Reply:
okay. ill add

Reply:
how many lines of output must be copied into doctring?

Reply:
I have added and fixed the jldoctests. They have to be spaced right to margin, no tabs before to be displayed like a code block